packages/manager/modules/cloud-connect/src/overview/datacenter/remove-datacenter/template.html

```diff
@@ -0,0 +1,19 @@
+<form
```

Comment:
**suggestion**: Same here than previous comment, why using a form ? And not directly use `oui-modal`'s `primary-action`